### ISSUE TITLE
Initial "Activating Patches" GUI message appears even when there are no enabled Patches.

### ISSUE THREAD

**Issue by nephros** [sailfishos-patches/patchmanager#361]
**SailfishOS VERSION** (Settings → About product → Build): 4.4.x
<br />**HARDWARE** (Settings → About product → Manufacturer & Product name): Xperia X10 and Xperia X10 III
<br />**PATCHMANAGER VERSION** (Settings → Patchmanager → \<Top pulley\> → About): 3.2.4
<br />

#### BUG DESCRIPTION

The Patchmanager App (i.e. the GUI started at Lipstick launch/after boot) does its thing even though it has nothing useful to do.

#### STEPS TO REPRODUCE

1. Have some patches installed
2. Disable all of them
3. Reboot
4. Lock code, security code entry, fresh Lipstick/Homescreen opens
5. Patchmanager GUI launches, displaying its remorse timer
6. After it has expired (or is swiped away), there were "0 patches activated"

#### ADDITIONAL INFORMATION

The final state of the GUI (i.e. the quit button) should be reached immediately if there are 0 activated patches.  
Maybe the GUI shouldn't launch at all.

**Comment by Olf0** [comment]
Is this really a bug or rather an "enhancement" from your POV?
As the message "0 patches activated" is correct *and* this is a rather unusual situation (why would one install Patchmanager, if not for using Patches?), I firmly believe it is not a bug and rather constitutes a potential beautification.  OTOH, this is a nice reminder, that one has Patchmanager installed (regardless if currently are no Patches enabled).

**Comment by Olf0** [comment]
TL;DR: It sure makes sense to omit the remorse timer, when there are no enabled Patches (to activate on boot).
Hence I concur with … 
> The final state of the GUI (i.e. the quit button) should be reached immediately if there are 0 activated patches.

As stated before, I think it makes sense to briefly show the GUI message (2 seconds is usually both, long enough to see and short enough to endure) even if there are no enabled Patches (to activate on boot).  A very simplistic approach would be to set the duration of the remorse timer to 2 seconds, if there is no enabled Patch to activate.

IMO to invest more time than for just that (above) into this GUI message is not worth the very little gain in usability; except it is fun. :wink:
